components/templates/NewsPage/index.tsx

```diff
@@ -9,9 +12,52 @@ const NewsPage: FunctionComponent<{
   data: PageEntry;
   locale: string;
 }> = async ({ data, section, locale }) => {
-  const { pressReleaseId } = data;
+  const { postType, pressReleaseId } = data;
+  const { slug } = postType[0];
 
-  if (pressReleaseId) {
+  // Announcements
+  if (slug !== "feature" && pressReleaseId) {
```

Comment:
add a second server data fetch to the news page if this is detected as an announcement